### PR TITLE
Made `process_nerfstudio_to_sdfstudio.py` include mono-priors

### PR DESCRIPTION
**Pull request by pablovela5620**
made it such that `process_nerfstudio_to_sdfstudio.py` will let you directly call `extract_monocular_priors.py`, along with adding a crop multiplier so that you can use a higher resolution image than 384 such as 768 (384*2) with depth/normal map upresing using simple interpolation. That way you can just call a single command, instead of 3 to generate the `meta_data.json` and relevant monocular priors

I'm not 100% if it helps a lot but typically polycam/colmap have much higher resolution images so it seems like it could be helpful. I haven't had the chance to do too many experiments to see if it results in higher fidelity meshes or not.

### CODE REVIEW

File: scripts/datasets/extract_monocular_cues.py [hunk 1]
```diff
@@ -152,23 +157,26 @@ def save_outputs(img_path, output_file_name):
...
-            # output = F.interpolate(output.unsqueeze(0), (512, 512), mode='bicubic').squeeze(0)
+            if scale_factor > 1:
+                output = F.interpolate(output.unsqueeze(0), scale_factor=scale_factor, mode="bicubic").squeeze(0)
             output = output.clamp(0, 1)
 
             np.save(save_path.replace(".png", ".npy"), output.detach().cpu().numpy()[0])
             plt.imsave(save_path, output.detach().cpu().squeeze(), cmap="viridis")
         else:
+            if scale_factor > 1:
+                output = torch.nn.functional.interpolate(output, scale_factor=scale_factor, mode="nearest")
```
Comment (niujinshuchong) [inline scripts/datasets/extract_monocular_cues.py:168]: Why use "nearest" for normal and "bicubic" for depth

File: scripts/datasets/process_nerfstudio_to_sdfstudio.py [hunk 2]
```diff
@@ -17,15 +19,23 @@ def main():
...
     parser.set_defaults(output_path="NONE")
 
-    parser.add_argument("--type", dest="type", default="colmap", choices=["colmap", "polycam"])
+    parser.add_argument("--type", dest="type", required=True, choices=["colmap", "polycam"])
     parser.add_argument("--indoor", action="store_true")
 
+    parser.add_argument("--crop-mult", dest="crop_mult", type=int, default=1)
+    parser.add_argument("--mono-prior", action="store_true")
+    parser.add_argument("--omnidata_path", dest="omnidata_path", help="path to omnidata model")
+    parser.set_defaults(omnidata_path="/home/pablo/0Dev/repos/omnidata/omnidata_tools/torch/")
```
Comment (niujinshuchong) [inline scripts/datasets/process_nerfstudio_to_sdfstudio.py:34]: Do you mind change the default value to "/home/yuzh/Projects/omnidata/omnidata_tools/torch/" so it's consistent with extract_monocular_cues.py

File: scripts/datasets/process_nerfstudio_to_sdfstudio.py [hunk 3]
```diff
@@ -17,15 +19,23 @@ def main():
...
+
+    parser.add_argument("--pretrained_models", dest="pretrained_models", help="path to pretrained models")
+    parser.set_defaults(pretrained_models="/home/pablo/0Dev/repos/omnidata/omnidata_tools/torch/pretrained_models/")
```
Comment (niujinshuchong) [inline scripts/datasets/process_nerfstudio_to_sdfstudio.py:37]: Same here "/home/yuzh/Projects/omnidata/omnidata_tools/torch/pretrained_models/"

**Review by niujinshuchong — COMMENTED** [review]
Thanks for the PR.

File: scripts/datasets/extract_monocular_cues.py (hunk 1 above)
Comment (pablovela5620) [reply scripts/datasets/extract_monocular_cues.py:168]: the depth value used bicubic from the original uncommented code, I tried using bicubic for normal map but it generated some artifacts around boundaries. I could also switch depth to nearest for consistency

**Review by niujinshuchong — APPROVED** [review]
Awesome. Thanks

**Pull request merged**